### ISSUE TITLE
public override func ambience(_ notification : Notification) method not called

### ISSUE THREAD

**Issue by roei46** [tmergulhao/Ambience#5]
Hi , i am trying implementing the above method and in contrast to the example you gave,  in my project it is not called but the color do change accordingly to what i fixed in the story board, any suggestions?  (using swift 4 and xcode 9.3  )
thanks in advanced !

**Comment by bpisano** [comment]
Hi,
I actually have the same problem. Everything should be correct. I have implemented `_ = Ambience.shared` in AppDelegate.
I have subclassed a UIButton. In `awakeFromNib()` I've set `ambience = true`. But `override func ambience(_ notification: Notification)` is never called. Using Xcode 9.3.1 and Swift 4.

**Comment by roei46** [comment]
yes... i have managed to do some walk-around but it is not the classiest solution and it has a lot of bugs.
Also i didnt got any answer from this dude so i am thinking on not use that pod.

**Comment by bpisano** [comment]
I'm looking for other solutions but it seems to be the easiest to use and that support IBDesignable. Maybe we should wait for iOS 12 🤷‍♂️

**Comment by roei46** [comment]
Wanna tell me about IBDesignable?
